.gitignore

```diff
@@ -9,6 +9,8 @@
 __pycache__/*
 .cache/*
 .*.swp
+bin/*
+lib/*
```

Comment:
This concerns multiple changes, so not repeating myself, but trying to give a detailed explanation once and referencing this later-on:

When going with something like https://www.gitignore.io/api/virtualenv they suggest `env`, `venv`, `.env` or `.venv`.
I prefer any of those over putting it in the main folder.
In the main folder your primarily expect package content and not runtime data, etc.

Reply:
Same story, deleted this from the repo's .gitignore and added them to my global one

Reply:
Well for the case of `virtualenv`: This _is_ generally a Python-specific thing, so IMHO it _is_ valid to add it to this file.
With this post I meant we should try to adhere to the standards, i.e. in this case `env`, `venv`, `.env` and/or `.venv`.

Generally, this repository covers https://www.gitignore.io/api/python,ansible,vagrant,virtualenv so that list may be a very good starting point if you want to improve upon what's already there.